core/cppgc.rs

```diff
@@ -36,11 +36,37 @@ pub fn make_cppgc_object<'a, T: GarbageCollected + 'static>(
   t: T,
 ) -> v8::Local<'a, v8::Object> {
   let state = JsRuntime::state_from(scope);
-  let templ =
-    v8::Local::new(scope, state.cppgc_template.borrow().as_ref().unwrap());
-  let func = templ.get_function(scope).unwrap();
-  let obj = func.new_instance(scope, &[]).unwrap();
+  let opstate = state.op_state.borrow();
+
+  // To support initializing object wraps correctly, we store the function
+  // template in the opstate during binding with `T`'s TypeId as the key
+  // because it'll be pretty annoying to propogate `T` everywhere.
```

Comment:
```suggestion
  // To initialize object wraps correctly, we store the function
  // template in OpState with `T`'s TypeId as the key when binding
  // because it'll be pretty annoying to propogate `T` generic everywhere.
```